Dnn.AdminExperience/ClientSide/Bundle.Web/package.json

```diff
@@ -24,6 +24,7 @@
     "babel-polyfill": "6.26.0",
     "create-react-class": "^15.6.3",
     "css-loader": "2.1.1",
+    "dayjs": "^1.11.1",
```

Comment:
We're including this via npm and also in the `contrib` folder, do we need both of these?

Reply:
Well, in this context yes, we could use the build to copy the file over but it is a bigger proposition.

Reply:
I just wanted to ensure we're using both approaches and not making copies needlessly. If they're both in use, that's fine (though de-duplicating would be a bonus)

Reply:
@bdukes I totally agree with you.  It is a real mess in here - mainly due to the use of `requirejs`.